### PR TITLE
whoami, winget, wmic: add Korean translation

### PR DESCRIPTION
**Pull request by Zamoca42**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.ko/windows/winget.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`winget upgrade --all`
+
+- 관리할 수 있는 모든 패키지 나열:
+
+`winget list --source winget`
+
+- 파일에서 패키지 가져오기 또는 설치된 패키지를 파일로 내보내기:
+
+`winget {{import|export}} {{--import-file|--output}} {{경로/의/파일}}`
```
Comment (kbdharun) [inline pages.ko/windows/winget.md:32]:
```suggestion
`winget {{import|export}} {{--import-file|--output}} {{경로/대상/파일}}`
```
Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md

Comment (Zamoca42) [reply pages.ko/windows/winget.md:32]: Apologies, that was my mistake. I will fix the typo related to that.

**Review by spageektti — APPROVED** [review]
Hi. Thanks for your contribution.

**Review by IMHOJEONG — COMMENTED** [review]
Thank you to everyone who confirmed.

I'll merge it right away

**Pull request merged**